packages/backend/seeds/seed_users.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.seed = async function (knex) {
+  // Deletes ALL existing entries
+  await knex('users').del()
+  await knex('users').insert([
+    {
+      id: 1,
```

Comment:
This will have to be an UUID. You can use [faker](https://fakerjs.dev/) to generate data if needed and [uuid](https://www.npmjs.com/package/uuid) for ID's.

Reply:
Done, but still have some issues with seeds. Will not use it for now